docs/best-practices.md

```diff
@@ -59,3 +59,51 @@ Apps _should_ allow all settings to be customized for each installation.
 ### Store configuration in the repository
 
 Any configuration _should_ be stored in the target repository. Unless the app is using files from an established convention, the configuration _should_ be stored in the `.github` directory. See the [API docs for `context.config`](https://probot.github.io/api/latest/classes/context.html#config).
+
+`context.config` supports sharing configs between repositories. If configuration for your app is not available in the target repository, it will be loaded from the `.github` directory of the target organization's `.github` repository.
```

Comment:
This seemed like the best place to talk about user-specific behavior. This is the sort of thing applications might need to reference about how to setup shared configuration. This is kind of a merging of the description in extensions and the README from probot-config. I didn't bring over  the "opt-in" and "opt-out" patterns because they seemed largely covered by the basic description.